### DISCUSSION TITLE
Reasons for using space instead of CSS grid?

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#1721] [General]
Hey!

I've only realised that 1.3 came with the new _space_ utilities.
Are there any reasons to use these instead of just using CSS grid to space between elements?

Thanks!

**Comment by lihbr** [comment]
Well if you want to support IE11 it makes sense I guess yes, also grid may be overkill sometimes when you just need to handle space between text elements like in a card or such~

**Comment by ThomOrlo** [comment]
I ask myself the same question, space and grid allow to have the same result…

- Horizontal space between children:
```css
<div class="flex space-x-4">
  <div>1</div>
  <div>2</div>
  <div>3</div>
</div>

<div class="inline-grid grid-flow-col gap-4">
  <div>1</div>
  <div>2</div>
  <div>3</div>
</div>
```

- Vertical space between children:
```css
<div class="space-y-4">
  <div>1</div>
  <div>2</div>
  <div>3</div>
</div>

<div class="inline-grid gap-4">
  <div>1</div>
  <div>2</div>
  <div>3</div>
</div>
```
